Tests/SourceControlTests/CheckoutManagerTests.swift

```diff
@@ -28,6 +28,9 @@ private class DummyRepository: Repository {
         fatalError("unexpected API call")
     }
 
+    func fetch() throws {
```

Comment:
Maybe we should check this is called where we expect it to be, since this is part of the API interface of this layer?